CONTRIBUTING.md

```diff
@@ -0,0 +1,73 @@
+# CONTRIBUTING
+
+Contributions are very welcome. When contributing code, please follow these
+simple guidelines.
+
+* Make sure you run `rustfmt` on your code. Also ensure all trailing whitespace is trimmed.
+* Run the tests with `cargo test --all`.
+* Before adding any code dependencies, check with the maintainers if this is okay.
+* Write properly formatted comments: they should be English sentences, eg:
+
+        // Return the current UNIX time.
+
+* Read the DCO and make sure all commits are signed off, using `git commit -s`.
```

Comment:
Does the DCO require outside contributors use their full legal (government) name for sign offs?